src/test/util/validation.cpp

```diff
@@ -11,6 +11,7 @@
 void TestChainState::ResetIbd()
 {
```

Comment:
bc1que90l5dy4yzn79x8xmem8t7dgn0c2zcqvxnr6x``
```suggestion
{0x0cA503AE9785a11e0ecFadB7fe98A57CC7893279
```